package.json

```diff
@@ -17,7 +17,8 @@
     "@actions/core": "^1.10.1",
     "@actions/exec": "^1.1.1",
     "@actions/github": "^6.0.1",
-    "@actions/tool-cache": "^2.0.1"
+    "@actions/tool-cache": "^2.0.1",
+    "semver": "^6.3.1"
```

Comment:
We were already transitively depending on this, explicitly added here to surface the dependency better